content/shmem_teams_intro.tex

```diff
@@ -70,32 +69,33 @@ \subsubsection*{Team Objects and Collective Ordering across \acp{PE}}
 
 \subsubsection*{Team Creation}
 
-Team creation is a collective operation on the parent team object. New teams
-result from a \FUNC{shmem\_team\_split\_*} routine, which takes a parent team
+New teams result from a \FUNC{shmem\_team\_split\_*} routine, which takes a parent team
 and other arguments and produces new teams that are a subset of the parent
-team. Teams that are created by a \FUNC{shmem\_team\_split\_*} routine may be
-provided a configuration argument that specifies attributes of each new team.
+team. Teams are created by a \FUNC{shmem\_team\_split\_*} routine that defines a
+collective operation over the set of all \acp{PE} which will be in the newly created
+teams. For each new team created by the split operation, a configuration argument
+may be provided to specify attributes of the new team.
 This configuration argument is of type \CTYPE{shmem\_team\_config\_t}, which
 is detailed further in Section~\ref{subsec:shmem_team_config_t}.
 
-As with any collective routine on a team, the program must ensure that there
-are no simultaneous split operations occurring on the same parent team on a
-given \ac{PE}, i.e. in separate threads.
-
-As with any collective routine on a team, team creation is matched across PEs based
-on ordering. So, team creation events must occur in the same order on all \acp{PE}
-in the parent team. Additionally, there must not be team creation
-operations from the same parent team simultaneously occurring that involve
-the same \acp{PE} in any resulting child teams.
-
-\begin{itemize}
-\item[] The following rule of practice will avoid any conflicts on team
-object resources during team creation:
-\item[] \emph{When a parent team is split multiple times, and the resulting child teams
-have overlapping membership, the program must call the \FUNC{shmem\_team\_sync}
-routine on the parent team between subsequent calls to split routines.}
-\end{itemize}
-
 Upon completion of a team creation operation, any resulting child teams will be
 immediately usable for any team-based operations, including creating new child teams,
 without any intervening synchronization.
```

Comment:
Change this text to specify where the intervening synchronization is or is not required. For example "no intervening synchronization on the child". Make this statement dovetail more smoothly in the subsequent description of requirements on the parent.